facade/instance.go

```diff
@@ -222,7 +222,7 @@ func (f *Facade) getInstance(ctx datastore.Context, hst host.Host, svc service.S
 		imgLogger.Debug("Loaded service image from registry")
 
 		return imageData.UUID == imageUUID, nil
-	}(svc.ImageID, state.ImageID)
+	}(svc.ImageID, state.ImageUUID)
```

Comment:
That change was in response to a request from Glenn in an earlier review:
tgjonestx 7 days ago
To avoid more confusion in the future, let's rename zkservice.ServiceState.ImageID to zkservice.ServiceState.ImageUUID